python/ray/tune/analysis/test_experiment_analysis.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import unittest
+import shutil
+import random
+import os
+import pandas as pd
+
+from experiment_analysis import ExperimentAnalysis
+import ray
+from ray.tune import run, sample_from, Trainable
+from ray.tune.examples.async_hyperband_example import MyTrainableClass
+from ray.tune.schedulers import AsyncHyperBandScheduler
+
+test_dir = "~/analysis_test"
```

Comment:
can you make these attributes of the ExperimentAnalysisSuite instead of globals?

Also, can the test_dir be a Tempdir? maybe try something like this to be cleaner - https://gist.github.com/odyniec/d4ea0959d4e0ba17a980

Reply:
@adizim one last comment here.